cli/templatepush.go

```diff
@@ -165,7 +168,24 @@ func (r *RootCmd) templatePush() *clibase.Cmd {
 		provisionerTags      []string
 		uploadFlags          templateUploadFlags
 		activate             bool
-		create               bool
+
+		displayName                    string
+		description                    string
+		icon                           string
+		requireActiveVersion           bool
+		disableEveryone                bool
+		defaultTTL                     time.Duration
+		failureTTL                     time.Duration
+		dormancyThreshold              time.Duration
+		dormancyAutoDeletion           time.Duration
+		maxTTL                         time.Duration
+		autostopRequirementDaysOfWeek  []string
+		autostopRequirementWeeks       int64
+		autostartRequirementDaysOfWeek []string
+		allowUserAutostart             bool
+		allowUserAutostop              bool
+		allowUserCancelWorkspaceJobs   bool
+		deprecationMessage             string
```

Comment:
I would expect all these variable declarations to live in a struct somewhere so we weren't constantly repeating them.